AndroidHomeApplication/app/src/main/java/com/example/androidhomeapplication/fragments/movieDetails/FragmentMovieDetails.kt

```diff
@@ -48,6 +46,22 @@ class FragmentMovieDetails : Fragment(R.layout.fragment_movie_details) {
         }
     }
 
+    private fun setResult(result: DataResult<Movie>) =
+        when (result) {
+            is DataResult.Loading -> {
```

Comment:
if no spinner than:
move those unhandled type to bottom(similar as we do with else branch)
in such cases u can use: 
is DataResult.Loading -> Unit
instead of
is DataResult.Loading -> {
}